include/zenoh/api/liveliness.hxx

```diff
@@ -13,23 +13,20 @@
 
 #pragma once
 
-#if defined(Z_FEATURE_UNSTABLE_API) && (defined(ZENOHCXX_ZENOHC) || Z_FEATURE_LIVELINESS == 1)
```

Comment:
#if  defined(ZENOHCXX_ZENOHC) || Z_FEATURE_LIVELINESS == 1 is still needed because of pico. Can be moved to higher level api.hxx though

Reply:
fixed in api.hxx